dev/src/generate_sql_schema.py

```diff
@@ -76,7 +78,9 @@ class GenerateCodeBlocks:
     )  # Key=Name, data: collected content of table
 
     @classmethod
-    def generate_the_code(cls) -> tuple[str, str, str, str, str, list[str], str, str]:
+    def generate_the_code(
+        cls,
+    ) -> tuple[str, str, str, str, str, list[str], str, str, list[str]]:
```

Comment:
Not specific to this PR, but this should really be a custom class/typed dict. Such a convoluted tuple is not understandable in any way